pypeit/spectrographs/vlt_xshooter.py

```diff
@@ -83,6 +83,26 @@ def compound_meta(self, headarr, meta_key):
             return parse.binning2string(binspec, binspatial)
         msgs.error("Not ready for this compound meta")
 
+    def config_independent_frames(self):
+        """
+        Define frame types that are independent of the fully defined
+        instrument configuration.
+
+        This method returns a dictionary where the keys of the dictionary are
+        the list of configuration-independent frame types. The value of each
+        dictionary element can be set to one or more metadata keys that can
+        be used to assign each frame type to a given configuration group. See
+        :func:`~pypeit.metadata.PypeItMetaData.set_configurations` and how it
+        interprets the dictionary values, which can be None.
+
+        Returns:
+            :obj:`dict`: Dictionary where the keys are the frame types that
+            are configuration-independent and the values are the metadata
+            keywords that can be used to assign the frames to a configuration
+            group.
+        """
+        return {'bias': 'arm', 'dark': 'arm'}
+
```

Comment:
I think it makes more sense for this to return None.  The only configuration key is arm meaning that the biases and dark shouldn't be treated differently from any other frame type.  You still need this method to overwrite the default in Spectrograph, but here you can just `return None` (I think).

Reply:
Based on the X-SHOOTER data that I have, this can't be `None` (incidentally, this is how I first noticed the issue!), and returning `arm` solves the issue in this case.

Reply:
I agree with @kbwestfall that, since `arm` is the only key in `configuration_keys`, putting it here is kind of redundant and also confusing, since the frames that go through this method are the one that are treated differently from the rest. I think `return {}` should work here, because if the dictionary is empty it means that there are no frames that are treated differently.

Reply:
Yes of course...  🤦‍♂️  I have updated this - thanks for the suggestion!